### ISSUE TITLE
train dist-s failed

### ISSUE THREAD

**Issue by stihuangyuan** [royalmelon0505/dist4d#14]
i follow your order process nuscense dataset, but whern i train dist-s  
      ./examples/scripts/mm_train_nus_768.sh
happen this error:
No such file or directory: '***/DiST_S/reproj_nus//train/frame_pesudo+4_mod'

**Comment by royalmelon0505** [comment]
The default data process code in DiST-S `.DiST_S/data_process/batch_reproj_nus_train_2frame.sh $start_idx $end_idx` only output *frame_pesudo+2*. You need to change `interval` ( L135) in `DiST_S/data_process/reproj_nus_scene_mp_2frame.py`. Therefore, you can get  *frame_pesudo-2* or *frame_pesudo+4*

**Comment by stihuangyuan** [comment]
> The default data process code in DiST-S `.DiST_S/data_process/batch_reproj_nus_train_2frame.sh $start_idx $end_idx` only output _frame_pesudo+2_. You need to change `interval` ( L135) in `DiST_S/data_process/reproj_nus_scene_mp_2frame.py`. Therefore, you can get _frame_pesudo-2_ or _frame_pesudo+4_

跑 ./examples/scripts/mm_train_nus_768.sh训练出现loss nan

08/28/2025 10:33:35 - INFO - __main__ - ***** Running training *****
08/28/2025 10:33:35 - INFO - __main__ -   Num examples = 11784
08/28/2025 10:33:35 - INFO - __main__ -   Num Epochs = 11
08/28/2025 10:33:35 - INFO - __main__ -   Instantaneous batch size per device = 1
08/28/2025 10:33:35 - INFO - __main__ -   Total train batch size (w. parallel, distributed & accumulation) = 4
08/28/2025 10:33:35 - INFO - __main__ -   Gradient Accumulation steps = 4
08/28/2025 10:33:35 - INFO - __main__ -   Total optimization steps = 30000
Steps:   0%|                                                                                                                                                                                                                              | 0/30000 [00:00<?, ?it/s]
loss tensor(nan, device='cuda:0', grad_fn=<MeanBackward0>)
Steps:   0%|     | 0/30000 [00:04<?, ?it/s, lr=5e-5, step_loss=nan]
loss tensor(nan, device='cuda:0', grad_fn=<MeanBackward0>)
Steps:   0%|     | 1/30000 [00:07<62:55:44,  7.55s/it, lr=5e-5, step_loss=nan]
loss tensor(nan, device='cuda:0', grad_fn=<MeanBackward0>)
Steps:   0%|     | 2/30000 [00:11<45:05:38,  5.41s/it, lr=5e-5, step_loss=nan]
